### PR TITLE
Remove support for running in Python 2

### PR DESCRIPTION
**Pull request by jgraham**
This just removes the support for building with Py2, it doesn't do anything about the (very) long tail of py2-isms in the code.

### CODE REVIEW

**Comment by denschub** [comment]
`requirements/dev-py2.txt` is probably something that can be removed as well. :)

**Comment by jgraham** [comment]
> `requirements/dev-py2.txt` is probably something that can be removed as well. :)

Good catch, thanks!

File: bin/run_docker_dev.sh [hunk 1]
```diff
@@ -31,11 +22,7 @@ if [[ $command == "build" ]]; then
     else
         TEST=0
     fi
-    if $PYTHON3; then
-        ARGS="--build-arg PYTHON_BINARY=python3 --build-arg PIP_BINARY=pip3"
-    else
-        ARGS="--build-arg PYTHON_BINARY=python2 --build-arg PIP_BINARY=pip2"
-    fi
+    ARGS=""
```
Comment (karlcow) [inline bin/run_docker_dev.sh:25]: You can safely remove `ARGS` if you also remove the value in

https://github.com/mozilla/wpt-sync/pull/795/files#diff-792e0f8b4e3285a392bb896d8f512eeeR52

**Review by karlcow — APPROVED** [review]
ok this is the devops part of removing python 2. 
That's cool. 

Do you plan to remove the python2-ism in future PRs?

**Pull request merged**